charts/posthog/templates/clickhouse-operator/configmap.yaml

```diff
@@ -127,9 +138,54 @@ data:
     
     # Max number of concurrent reconciles in progress
     reconcileThreadsNumber: 10
-    reconcileWaitExclude: false
+    reconcileWaitExclude: true
```

Comment:
Do you know if this change is needed?

Reply:
It's coming from upstream and I think it is needed